opensrp-giz-malawi/src/main/java/org/smartregister/giz/util/PdfGeneratorHelper.java

```diff
@@ -71,7 +71,7 @@ public static ContextStep getBuilder() {
             }
 
         } catch (Exception e) {
-            Timber.e(e.getMessage());
+            Timber.e("Could not inflate Layout");
```

Comment:
Could this be `Timber.e(e, "Could not inflate Layout");`

Reply:
Done